extensions/amp-story/0.1/_locales/default.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {LocalizedStringBundleDef, LocalizedStringId} from '../localization';
+
+
+/**
+ * Localized string bundle used for English strings.
+ * @const {!LocalizedStringBundleDef}
+ */
+export default {
+  [LocalizedStringId.AMP_STORY_EXPERIMENT_ENABLE_BUTTON_LABEL]: {
+    string: 'Enable',
+    description: 'Label for a button that enables the amp-story experiment.',
+  },
+  [LocalizedStringId.AMP_STORY_EXPERIMENT_ENABLED_TEXT]: {
+    string: 'Experiment enabled.  Please reload.',
+    description: 'Text that is shown once the amp-story experiment has ' +
+        'been successfully enabled.',
+  },
+  [LocalizedStringId.AMP_STORY_SHARING_CLIPBOARD_FAILURE_TEXT]: {
+    string: ':(',
+    description: 'String shown in a toast to inform the user that a link ' +
```

Comment:
Nit: a translator would probably not know what a toast is

Reply:
Updated!